redisinsight/ui/src/components/base/inputs/SwitchInput.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import { Switch } from '@redis-ui/components'
+
+const SwitchInput: React.FC<React.ComponentProps<typeof Switch>> = ({
+  style,
+  ...props
+}) => (
```

Comment:
So far we follow the pattern to define props above the component and type them in the arguments list
```suggestion
type SwitchInputProps = React.ComponentProps<typeof Switch
const SwitchInput = ({
  style,
  ...props
}: SwitchInputProps) => (
```